plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -17,10 +17,13 @@
 import io.airlift.configuration.ConfigDescription;
 import io.airlift.configuration.LegacyConfig;
 
+import javax.validation.constraints.Max;
 import javax.validation.constraints.Min;
 
 public class JdbcMetadataConfig
 {
+    static final int MAX_ALLOWED_INSERT_BATCH_SIZE = 1_000_000;
```

Comment:
Putting the limit here seems reasonable. Please suggest better place where to put it.